### PR TITLE
Upgrade all packages to latest versions

### PR DESCRIPTION
**Pull request by samblackk**
Step 1 for https://github.com/astronomer/issues/issues/1312 

This PR upgrades all of the packages in `package.json` to their latest stable versions.

@andriisoldatenko Test this branch against the CLI.
@ryanahamilton Test this branch against the UI. 

CC @ryw @vishwas-astro

### CODE REVIEW

File: src/lib/oauth/config/index.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import config from "config";
 import shortid from "shortid";
 import { has, merge, get, upperFirst } from "lodash";
 
-export const providerCfg = config.get("auth.openidConnect");
+export const providerCfg = Object.create(config.get("auth.openidConnect"));
```
Comment (samblackk) [inline src/lib/oauth/config/index.js:8]: The only fix needed after updating the packages.

**Comment by andriisoldatenko** [comment]
@samblackk does it work practically? should we try run it in cluster?

**Comment by samblackk** [comment]
@andriisoldatenko It runs locally, and docker builds locally with no issues. Unit tests pass locally. Waiting for CI to wrap up to see if there's anything else.

**Comment by andriisoldatenko** [comment]
`npm run lint`

**Comment by samblackk** [comment]
@andriisoldatenko lint fix pushed

**Comment by samblackk** [comment]
🙄these unit tests passed locally, but failed in CI

**Comment by ryw** [comment]
Gets us down from `critical - 8, high - 105` to `critical - 7, high - 70` - so progress :)

**Comment by ryanahamilton** [comment]
I haven't debugged yet, but getting a GraphQL permissions error on the `appConfig` query from the login screen so far.

**Comment by samblackk** [comment]
Closing this. Caused too many problems. It's reported that 73 of 77 CVEs from Twistlock are false positives. Will only be upgraded the effected packages in a new PR.

**Pull request closed without merging**